markdown/usage/installation.md

```diff
@@ -31,125 +18,52 @@ java -version
 ```
 
-See [nextflow.io](https://www.nextflow.io/) for further instructions on how to install and configure Nextflow.
+We recommend using a personal installation of Nextflow where possible, instead of using a system-wide installation. This makes it easier to update.
 
-## Install the pipeline
+Updating nextflow is as simple as running `nextflow self-update`.
 
-### Automatic
-This pipeline itself needs no installation - nextflow will automatically fetch it from GitHub if `nf-core/YOUR_PIPELINE` is specified as the pipeline name.
+Once installed you will probably need to configure Nextflow to run on your system. For instructions, see [_Nextflow configuration_](configuration).
 
-### Offline
-The above method requires an internet connection so that Nextflow can download the pipeline files. If you're running on a system that has no internet connection, you'll need to download and transfer the pipeline files manually:
-
-```bash
-wget https://github.com/nf-core/YOUR_PIPELINE/archive/master.zip
-mkdir -p ~/my-pipelines/nf-core/
-unzip master.zip -d ~/my-pipelines/nf-core/
-cd ~/my_data/
-nextflow run ~/my-pipelines/nf-core/YOUR_PIPELINE-master
-```
 
-To stop nextflow from looking for updates online, you can tell it to run in offline mode by specifying the following environment variable in your ~/.bashrc file:
+## Pipeline software
 
-```bash
-export NXF_OFFLINE='TRUE'
-```
-
-### Development
-
-If you would like to make changes to the pipeline, it's best to make a fork on GitHub and then clone the files. Once cloned you can run the pipeline directly as above.
-
-
-## Pipeline configuration
-By default, the pipeline loads a basic resource configuration:
-
-```nextflow
-process {
-
-  cpus = { check_max( 2, 'cpus' ) }
-  memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  time = { check_max( 2.h * task.attempt, 'time' ) }
-
-  errorStrategy = { task.exitStatus in [143,137,104,134,139] ? 'retry' : 'terminate' }
-  maxRetries = 1
-  maxErrors = '-1'
-
-  // Process-specific resource requirements
-  withName: fastqc {
-    time = { check_max( 8.h * task.attempt, 'time' ) }
-  }
-  withName:toolX {
-    cpus = { check_max( 8, 'cpus' ) }
-    memory = { check_max( 8.GB * task.attempt, 'memory' ) }
-  }
-  withName: toolY {
-    cpus = { check_max( 10, 'cpus' ) }
-    memory = { check_max( 200.GB * task.attempt, 'memory' ) }
-    time = { check_max( 5.h * task.attempt, 'time' ) }
-  }
-}
-
-params {
-  // Defaults only, expecting to be overwritten
-  max_memory = 128.GB
-  max_cpus = 16
-  max_time = 240.h
-  igenomes_base = 's3://ngi-igenomes/igenomes/'
-}
-```
+An analysis pipeline chains the execution of multiple tools together.
+Historically, all tools would have to be manually installed, often a source of great frustration and a key step where reproducibility between analyses is lost.
+All nf-core pipelines fully utilise the built-in support for software packaging that Nextflow offers - namely Docker, Singularity and Conda:
```

Comment:
This isnt entirely true. Some pipelines like `imcyto` dont offer all of these options because it impossible to create a Conda environment with all of the pipeline dependencies.